src/CONST.ts

```diff
@@ -653,6 +653,7 @@ const CONST = {
                     LEAVE_ROOM: 'LEAVEROOM',
                     UPDATE_ROOM_DESCRIPTION: 'UPDATEROOMDESCRIPTION',
                 },
+                UNAPPROVED: 'UNAPPROVED',
```

Comment:
Also we discussed putting this at the bottom, and putting a comment above noting that "any action below is an old, historical report action that we just want to show in plain text for now" - or something like that